Changes

```diff
@@ -109,6 +109,11 @@ Working version
 - GPR#1966: Add Format semantic tags using extensible sum types.
   (Gabriel Radanne, review by Nicolás Ojeda Bär)
 
+- GPR#1794: Add arithmetic operators +., -.,... and functions
```

Comment:
"Add arithmetic operators" to be dropped

Reply:
Well spotted!

Reply:
This is fixed in the new commits below.